CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Release History
 
+## 1.14.0 (TBD)
+
+### Improvements
+
+- Added a new example for the split function using `lit()` to specify a string separator.
+
```

Comment:
we usually don't add a changelog for a small change like this. You may use label `NO-CHANGELOG-UPDATES`

```suggestion
```